src/traces/choropleth/plot.js

```diff
@@ -122,9 +106,11 @@ function style(geo) {
             )
         );
 
-        s.selectAll('path.choroplethlocation').each(function(pt) {
+        s.selectAll('path.choroplethlocation').each(function(_, i) {
+            var pt = calcTrace[i];
```

Comment:
why is`pt` not just the first arg?

Reply:
done in https://github.com/plotly/plotly.js/pull/2030/commits/71c535541bb4816310f782ce9ee836247cf54c78